mne/preprocessing/nirs/_beer_lambert_law.py

```diff
@@ -18,15 +18,15 @@
 from ..nirs import _validate_nirs_info, source_detector_distances
 
 
-def beer_lambert_law(raw, ppf=6.0):
+def beer_lambert_law(raw, ppf=(6.0, 6.0)):
     r"""Convert NIRS optical density data to haemoglobin concentration.
 
     Parameters
     ----------
     raw : instance of Raw
         The optical density data.
-    ppf : float
-        The partial pathlength factor.
+    ppf : tuple
```

Comment:
Need to keep backward compat so...
```suggestion
    ppf : tuple | float
```